eq-author/src/components/Validation/Date/DatePreview.js

```diff
@@ -45,9 +44,10 @@ const DatePreview = ({
   return (
     <React.Fragment>
       <div>
-        {`${value} ${unit.toLowerCase()} ${relativePosition.toLowerCase()}:`}
+        {`${value} ${unit.toLowerCase()} `}
+        {relativePosition && `${relativePosition.toLowerCase()}:`}
       </div>
-      <div>{rule[entityType]}</div>
+      <div>{entityType && rule[entityType]}</div>
```

Comment:
Do we want an empty div when its not needed?

Reply:
No. Ideally, we should keep DOM elements to a minimum.  I'll rework this.